src/overlays/actors/ovl_En_Lift_Nuts/z_en_lift_nuts.c

```diff
@@ -61,32 +61,55 @@ ActorInit En_Lift_Nuts_InitVars = {
-    { &object_dnt_Anim_00577C, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
-    { &object_dnt_Anim_004E38, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
-    { &object_dnt_Anim_0029E8, 1.0f, 4.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
-    { &object_dnt_Anim_0029E8, 1.0f, 8.0f, 0.0f, ANIMMODE_ONCE, 0.0f },
+typedef enum {
+    /*  0 */ ENLIFTNUTS_ANIM_STANDING,
+    /*  1 */ ENLIFTNUTS_ANIM_WALK,
+    /*  2 */ ENLIFTNUTS_ANIM_RISE_UP,
+    /*  3 */ ENLIFTNUTS_ANIM_BURROW_LONG,
+    /*  4 */ ENLIFTNUTS_ANIM_EXCITED_START,
+    /*  5 */ ENLIFTNUTS_ANIM_EXCITED_LOOP,
+    /*  6 */ ENLIFTNUTS_ANIM_EXCITED_END,
+    /*  7 */ ENLIFTNUTS_ANIM_TAKE_OFF_HAT,
+    /*  8 */ ENLIFTNUTS_ANIM_FLY_START,
+    /*  9 */ ENLIFTNUTS_ANIM_FLY_LOOP,
+    /* 10 */ ENLIFTNUTS_ANIM_SHOCKED_START,
+    /* 11 */ ENLIFTNUTS_ANIM_SHOCKED_SHAKE_HEAD,
+    /* 12 */ ENLIFTNUTS_ANIM_SHOCKED_POUND,
+    /* 13 */ ENLIFTNUTS_ANIM_SHOCKED_END,
+    /* 14 */ ENLIFTNUTS_ANIM_THINK,
+    /* 15 */ ENLIFTNUTS_ANIM_BOB,
+    /* 16 */ ENLIFTNUTS_ANIM_JUMP,
+    /* 17 */ ENLIFTNUTS_ANIM_BURROW_HALF,
+    /* 18 */ ENLIFTNUTS_ANIM_BURROW_SHORT,
+    /* 19 */ ENLIFTNUTS_ANIM_MAX
+} EnLiftNutsAnimation;
+
+static AnimationInfo sAnimations[ENLIFTNUTS_ANIM_MAX] = {
+    { &gBusinessScrubStandingAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
+    { &gBusinessScrubWalkAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
+    { &gBusinessScrubRiseUpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
+    { &gBusinessScrubBurrowAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
+    { &gBusinessScrubExcitedStartAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },
+    { &gBusinessScrubExcitedLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
+    { &gBusinessScrubExcitedEndAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },
+    { &gBusinessScrubTakeOffHatAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
+    { &gBusinessScrubFlyStartAnim, 0.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
+    { &gBusinessScrubFlyLoopAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
+    { &gBusinessScrubShockedStartAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -2.0f },
+    { &gBusinessScrubShockedShakeHeadAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
+    { &gBusinessScrubShockedPoundAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
+    { &gBusinessScrubShockedEndAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },
+    { &gBusinessScrubThinkAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
+    { &gBusinessScrubBobAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },
+    { &gBusinessScrubJumpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
+    { &gBusinessScrubBurrowAnim, 1.0f, 4.0f, 0.0f, ANIMMODE_ONCE, -4.0f },
+    { &gBusinessScrubBurrowAnim, 1.0f, 8.0f, 0.0f, ANIMMODE_ONCE, 0.0f },
 };
 
-Gfx* D_80AEBF18[] = {
-    object_dnt_DL_008290,
-    object_dnt_DL_001350,
-    object_dnt_DL_001420,
+Gfx* sEyeTextures[] = {
+    gBusinessScrubEyesDL,
+    gBusinessScrubEyesWideDL,
+    gBusinessScrubEyesSquintDL,
     NULL,
 };
```

Comment:
```suggestion
Gfx* sEyeTextures[] = {
    gBusinessScrubEyesDL,
    gBusinessScrubEyesWideDL,
    gBusinessScrubEyesSquintDL,
};

static s32 sPad = 0;
```
I think this would be more appropriate?